.eslintrc

```diff
@@ -0,0 +1,205 @@
+env:
+  es6: true
```

Comment:
might be useful to pull in `plugins` https://github.com/mozilla/fxa-content-server/blob/master/.eslintrc#L2 so you are not repeating the rules, if any rules are missing that you want add them to the fxa plugin